sample-app/src/components/cards/StandardCard.tsx

```diff
@@ -14,28 +47,62 @@ export interface StandardCardProps extends CardProps {
  * @param props - An object containing the result itself.
  */
 export function StandardCard(props: StandardCardProps): JSX.Element {
-  const { configuration, result } = props;
+  const { configuration, result, customCssClasses, compositionMethod } = props;
+  const cssClasses = useComposedCssClasses(builtInCssClasses, customCssClasses, compositionMethod);
 
-  return (
-    <div className='StandardCard'>
-      {configuration.showOrdinal && result.index && renderOrdinal(result.index)}
-      <div className='StandardCard__contentWrapper'>
-        {result.name && renderTitle(result.name)}
+  const cta1 = result.rawData.c_primaryCTA as CtaData;
```

Comment:
Should we have a typeGuard function to check if it follows CtaData interface (log a message if it doesn't) instead of using force type assertion 'as'?

Reply:
Sure